### PR TITLE
April 2022 Steering Council community update.

### PR DESCRIPTION
**Pull request by gpshead**
Summarizing our notes from April before PyCon.

### CODE REVIEW

File: updates/2022-04-steering-council-update.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+## 2022-04-25
+
+* Discussed GitHub Issues migration administrivia.
+* Discussed our impending PyCon keynote.
+* Discussed how most of us need to reread PEP-687.
+* Discussed what role the python-ideas mailing list has.
+* Discussed what topics we should bring up at the language summit.
+* Discussed how we'd like to see PEPs discussed.
+* Discussed release managment team woes and bus factors.
```
Comment (JelleZijlstra) [inline updates/2022-04-steering-council-update.md:73]:
```suggestion
* Discussed release management team woes and bus factors.
```

File: updates/2022-04-steering-council-update.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
+# SC meeting summary - April 2022
+
+## 2022-04-04
+
+* Discussed CPython public C APIs with underscores in them as
+  [PEP-523](https://peps.python.org/pep-0523/), for example, added vs a desire
+  for perhaps a stable only within bugfixes API tier.
```
Comment (bskinn) [inline updates/2022-04-steering-council-update.md:7]: Coming from outside the SC's discussion on this, this bullet is very hard to parse. If it's meant mainly for SC reference, then no change may be needed. If it's meant for general/public information, I recommend clarification.

1. What was/might be added? A PEP 523 frame evaluation API? How does PEP 523 relate to APIs with underscores in them? APIs with underscores as opposed to what?

2. Does "stable only within bugfixes API tier" mean "an API tier for things that are stable only within bugfixes"? If so, hyphenating might improve clarity: "... a desire for perhaps a stable-only-within-bugfixes API tier." (Presumably this is the same as the 'semi-stable' API mentioned lower down -- if so, perhaps make that forward reference here?)

Comment (encukou) [reply updates/2022-04-steering-council-update.md:8]: I wrote the main result of this discussion up as a PEP, maybe noting that will reduce the confusion?
```suggestion
  for perhaps a stable only within bugfixes API tier.
  [Note from the future: this discussion later led to [PEP-689](https://peps.python.org/pep-0689/)
  (Unstable C API tier).]
```

Comment (bskinn) [reply updates/2022-04-steering-council-update.md:8]: Yes, that definitely helps my (2) -- but (1) is still unresolved.

Is PEP 689 an alternative approach to solving the same thing that PEP 523 aims to solve? They seem quite different to me, but I'm unfamiliar with the C API.

Or -- is the (eventual) PEP 689 something that emerged out of a subset of the discussion around PEP 523?

Comment (gpshead) [reply updates/2022-04-steering-council-update.md:8]: Draft PEP 689 definitely emerged from all the discussions around this. See also https://github.com/python/steering-council/issues/123 which immediately preceeded the PEP 689 creation.

PEP 523 is a 6 year old PEP, it was being cited as an example of intentionally public APIs intentionally containing underscores from the past. (a pre steering council even existing decision)

**Review by brettcannon — APPROVED** [review]

**Comment by bskinn** [comment]
@gpshead In case it didn't send a notification -- please see my suggested edit in a PR on your fork.

**Comment by gpshead** [comment]
> @gpshead In case it didn't send a notification -- please see my suggested edit in a PR on your fork.

A PR to my PR branch in my fork? LOL no I didn't notice that. Thanks, merged into this PR.

**Comment by bskinn** [comment]
Yeah. I only learned yesterday about the "suggested change" feature. Much easier/faster/better. 😅
